service/cloudconfig/service.go

```diff
@@ -75,10 +75,43 @@ func New(config Config) (*CloudConfig, error) {
 func (c CloudConfig) NewMasterCloudConfig(customObject providerv1alpha1.AzureConfig) (string, error) {
 	params := k8scloudconfig.Params{
 		Cluster: customObject.Spec.Cluster,
+		Hyperkube: k8scloudconfig.Hyperkube{
+			Apiserver: k8scloudconfig.HyperkubeApiserver{
+				BindAddress: "0.0.0.0",
+				HyperkubeDocker: k8scloudconfig.HyperkubeDocker{
+					RunExtraArgs: []string{
+						"-v /etc/kubernetes/config:/etc/kubernetes/config",
+						"-v /var/lib/waagent/ManagedIdentity-Settings:/var/lib/waagent/ManagedIdentity-Settings:ro",
+					},
+					CommandExtraArgs: []string{
+						"--cloud-config=/etc/kubernetes/config/azure.yaml",
+					},
+				},
```

Comment:
This one is nested and it looks kind of ugly. I proposed improvement here: https://github.com/giantswarm/k8scloudconfig/pull/271